### PR TITLE
Add rich quirk (maybe balance needed)

### PR DESCRIPTION
**Pull request by NeonNik2245**
## About The Pull Request

Adds rich quirk (4 points)

image

This give you your very special credit card, it has random amount of credits at the shift start (now it's 1000-5000 credits)

image

You can freely transfer credits on and from your credit card.

## How This Contributes To The Nova Sector Roleplay Experience

People would be able to play rich characters

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

image  
</details>

## Changelog

:cl:
add: adds rich quirk and credit cards.
/:cl:

### CODE REVIEW

File: code/datums/quirks/positive_quirks/rich.dm [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+#define MIN_CREDIT_ON_CARD 1000
+
+/datum/quirk/item_quirk/rich
+	name = "Rich"
+	desc = "You have really much money. So much, that you ever take your credit card with you. Be sure you won't lose it."
+	gain_text = span_danger("You feel yourself rich!")
+	lose_text = span_notice("You suddenly feel yourself poor.")
+	medical_record_text = "Patient has a large amount of money."
+	icon = FA_ICON_COINS
+	value = 4
```
Comment (SomeRandomOwl) [inline code/datums/quirks/positive_quirks/rich.dm:11]:
```suggestion
/datum/quirk/item_quirk/rich
	name = "Rich"
	desc = "You have a lot of money. So much, that you take your credit card with you. Be sure you don't lose it."
	gain_text = span_danger("You feel come across a credit card on the floor!")
	lose_text = span_notice("You are bankrupt! Your credit card account has been closed.")
	medical_record_text = "Patient has a large amount of money."
	icon = FA_ICON_COINS
	value = 4
```

**Review by SomeRandomOwl — CHANGES_REQUESTED** [review]
Ok, I am not sure what i feel about this PR. I will have to have discussions about it. 

Regardless, grammar pass.

File: code/datums/quirks/positive_quirks/rich.dm (hunk 1 above)
Comment (CliffracerX) [reply code/datums/quirks/positive_quirks/rich.dm:11]: Should be "you come across", not "you feel come across"

Comment (SomeRandomOwl) [reply code/datums/quirks/positive_quirks/rich.dm:11]: shush

**Comment by Ghommie** [comment]
Make sure this is incompatible with indebted.

**Comment by NeonNik2245** [comment]
I can change amount of money (min and max) and make this quirk more expensive. I just need your help to determine those numbers

**Comment by NeonNik2245** [comment]
> Make sure this is incompatible with indebted, or not I guess.

Yeah, I'll add it

**Comment by NeonNik2245** [comment]
> > I can change amount of money (min and max) and make this quirk more expensive (like 8 points). I just need your help to determine those numbers
> 
> The problem is that we don't have quirk points. We turned them off! Downstreams might use them, still, and prices are relevant to them, but a quirk just goes into a slot irregardless. None of the other positive quirks barring arguably freerunner has a bombastic mechanical effect, unlike this.

We - NovaSector?

**Comment by GoldenAlpharex** [comment]
After discussion, we're not really interested in this quirk, sorry.

**Pull request closed without merging**